test/libyul/yulInterpreterTests/hex_literals.yul

```diff
@@ -0,0 +1,9 @@
+{
+	let x := hex"1122334455667788990011223344556677889900"
```

Comment:
Can you also include characters from `a-f`.

Reply:
Also do we have tests validating the literal? What about something like `hex"abc"`?

Reply:
Added https://github.com/ethereum/solidity/pull/11173/files#diff-676a80d4a8d72728931f536c17bed78279e92afb6e27bce48c309c748cbacd0f